.github/workflows/build-pull-request.yml

```diff
@@ -69,7 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     if: always()
     steps:
-      - if: >
+      - if: |
```

Comment:
this change is just for consistency, as `>` multi-line doesn't seem to work as well for some if expressions